network_acls.tf

```diff
@@ -3,10 +3,10 @@
 ##############################################################################
 
 locals {
-  cluster_rules = [
-    # Cluster Rules
+  below_the_line_rules = [
```

Comment:
I agree that if you never heard of "below_the_line", it might be confusing.  Most customers hear "private network" or "ibm internal".

Reply:
agree that below the line is not an appropriate label because it doesn't describe to all users what it refers to. Any of the choices that @vburckhardt or @argeiger listed seem fine to me.

Reply:
Thanks for the feedback, going with "ibm_cloud_internal_rules"